README.md

```diff
@@ -59,5 +59,5 @@ Contributions are warmly welcomed and greatly appreciated. Here are a few ways y
 In case you have any problems when using OceanBase Database, welcome reach out for help:
 
 - [GitHub Issue](https://github.com/oceanbase/oceanbase/issues)
-- [Official Website](https://open.oceanbase.com/)
-- Knowledge base [link TODO]
+- [Official Website](https://www.oceanbase.com/community/answer)
```

Comment:
the link here is not available.
Thanks for your feecback.

Reply:
The link is available, but I realize that  it maybe isn't correct link.
There is a problem: the forum is for the 'oceanbase' instead of 'open.oceanbase'.
So, does the link of forum of discussion also should change into 'TODO'?

Reply:
Thanks for your feedback.
The official Website is correct, because the oceanbase.com is the business version website and open.oceanbase.com is the free open version website.
The link of discussion I think is note the knowledge base document, we will update the document as soon as possible.